### DISCUSSION TITLE
Unable to Save Files in the Filesystem

### DISCUSSION THREAD

**Discussion by rama-kairi** [blitz-js/blitz#1192] [Q&A]
Is there any Documentation available to Save Files in the Filesystem instead of pushing it to any CDN.

**Comment by flybayer** [comment]
@digital-rama currently no! But you can do this with [a regular API route](https://blitzjs.com/docs/api-routes).

We also have an open issue for doing it via mutations: https://github.com/blitz-js/blitz/issues/843

**Reply by hardfire** [reply]
Here's an example of how i do it using a regular API route - https://gist.github.com/hardfire/4c69b88ce62e4243a35e998ddcac08e1
Pulled everything into one file to create a contained example. I hope it helps.

**Reply by vonphlpp** [reply]
Nice after 5 h of trying hard to upload a image I found your link. Now it is working. THX!
